samples/OneDrivePhotoBrowser/project.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "dependencies": {
     "Microsoft.NETCore.UniversalWindowsPlatform": "5.0.0",
-    "Newtonsoft.Json": "7.0.1"
+    "Newtonsoft.Json": "6.0.2"
```

Comment:
Need isn't quite the right word. The idea is to use the lowest possible version we need instead of forcing developers into a higher version. Since nugget uses >= versioning devs can upgrade to the latest and greatest only if they want to.